src/useCreatContentPair.ts

```diff
@@ -1,26 +1,27 @@
-import { useEffect, useState } from "react";
+import React from "react";
 import { createDecoder, createEncoder } from "@waku/core";
 import type { Decoder, Encoder } from "@waku/core/dist/lib/message/version_0";
 
-type ContentPair = {
-  encoder: null | Encoder;
-  decoder: null | Decoder;
-};
+import type { ContentPair } from "./types";
 
 /**
  * Creates Encoder / Decoder pair for a given contentTopic.
  * @param {string} contentTopic - topic to orient to
- * @param {boolean} ephemeral - optional, makes messages ephemeral
+ * @param {boolean} ephemeral - makes messages ephemeral, default to false
  * @returns {Object} Encoder / Decoder pair
  */
-export const useContentPair = (
+export const useCreateContentPair = (
```

Comment:
As discussed, I think we should have one hook/component to create a decoder and one to create an encoder as they rarely go in pair, web-chat is the exception to the rule.

Reply:
Added as follow up item